src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/errormessage/ErrorMessage.scala

```diff
@@ -25,11 +26,24 @@ final case class ErrorMessage(
   id: Option[String] = None,
   classes: String = "",
   attributes: Map[String, String] = Map.empty,
-  visuallyHiddenText: Option[String] = Some("Error"),
+  visuallyHiddenText: Option[String] = Some(ErrorMessage.defaultVisuallyHiddenText),
   content: Content = Empty
 )
 
 object ErrorMessage {
+  
+  val defaultVisuallyHiddenText = "Error"
+
+  implicit class I18nErrorMessage(errorMessage: ErrorMessage)(implicit messages: Messages) {
```

Comment:
As discussed, agreed that rather than having the pattern of create with English language then copy and translate, the translation of the `visuallyHiddenText` parameter would happen prior to construction, i.e. within the `ImplicitsSupport` class.

Reply:
raised a new PR for the different approach https://github.com/hmrc/play-frontend-govuk/pull/105